pkg/kotsclient/cluster_create.go

```diff
@@ -79,6 +85,13 @@ func (c *VendorV3Client) CreateCluster(opts CreateClusterOpts) (*types.Cluster,
 		Tags:                   opts.Tags,
 	}
 
+	if opts.MinNodeCount >= 0 {
```

Comment:
Sorry I think i steered you in the wrong direction here. This will unexpectedly drop a value if its invalid. I actually think you need to do some validation in the runners.createCluster function.

ideally what id like to see in that function is 

```
if r.args.createClusterMinNodeCount >= 0 {
  opts.MinNodeCount = r.args.createClusterMinNodeCount
} else if r.args.createClusterMinNodeCount != -1 {
  return errors.New("invalid value for flag --min-node-count (valid values are ...)", )
}
```